plugin/src/main/java/app/cash/paraphrase/plugin/GenerateFormattedResources.kt

```diff
@@ -63,7 +63,10 @@ internal abstract class GenerateFormattedResources @Inject constructor() : Defau
     //   values-es -> [strings.xml]
     val filesByConfiguration = valuesFolders
       .associate { folder ->
-        ResourceFolder(folder.name) to folder.listFiles().orEmpty().toList()
+        ResourceFolder(folder.name) to folder.listFiles()
+          .orEmpty()
+          .filter { it.extension.equals("xml", ignoreCase = true) }
+          .toList()
```

Comment:
This is redundant now as `filter` already returns a list.

Reply:
Good catch, fixed in #285